src/rider/main/kotlin/com/jetbrains/rider/plugins/unreal/actions/PlaySettingsActions.kt

```diff
@@ -8,13 +8,15 @@ import com.jetbrains.rider.plugins.unreal.UnrealHost
 
 class PlaySettings : DefaultActionGroup() {
     override fun update(e: AnActionEvent) {
+        super.update(e)
```

Comment:
update for `DefaultActionGroup` does nothing, it was skipped on purpose

Reply:
Discussed in private about conservative approach